tests/jupyterhub_api/SessionUtils.py

```diff
@@ -0,0 +1,48 @@
+import os
+import subprocess
+import requests
+import json
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+import yaml
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+
+class Session:
```

Comment:
Without documentation/docstrings is hard to understand why you need this class and what it does.